### PR TITLE
feat: Store WGS raw plots

### PR DESCRIPTION
**Pull request by ivadym**
### Added:
- Store WGS CNV report plots

Hermes changes: https://github.com/Clinical-Genomics/hermes/pull/86/commits/a55e390e5f6c5d36beed633032416a687acbbd7a

### Review and tests: 
- [ ] Tests pass
- [ ] Code review
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

File: BALSAMIC/snakemake_rules/variant_calling/somatic_sv_tumor_normal.rule [hunk 1]
```diff
@@ -194,13 +194,13 @@ rule ascat_tumor_normal:
     output:
         final_vcf = vcf_dir + "CNV.somatic." + config["analysis"]["case_id"] + ".raw.ascat.vcf.gz",
         ascat_copynumber = vcf_dir + "CNV.somatic." + config["analysis"]["case_id"] + ".ascat.copynumber.txt.gz",
-        sample_statistics = temp(vcf_dir + "CNV.somatic." + config["analysis"]["case_id"] + ".ascat.samplestatistics.txt"),
```
Comment (mathiasbio) [inline BALSAMIC/snakemake_rules/variant_calling/somatic_sv_tumor_normal.rule:197]: I wonder if this temp() solution is working, I think these files were saved anyway? Regardless it's nice to remove the temp() call, it's just confusing to keep it if it's not doing anything :D

Comment (mathiasbio) [reply BALSAMIC/snakemake_rules/variant_calling/somatic_sv_tumor_normal.rule:197]: Or was this temp() was used in the delivery function of balsamic to not add these files to the hk-file? And by removing it we're now adding these to there, and then we need the corresponding change in hermes to start to store them? : )

File: BALSAMIC/constants/rules.py [hunk 2]
```diff
@@ -204,6 +204,7 @@
     "delly_cnv_tumor_only",
     "delly_cnv_tumor_normal",
     "ascat_tumor_normal",
+    "cnvpytor_tumor_only",
```
Comment (mathiasbio) [inline BALSAMIC/constants/rules.py:207]: And it makes sense to add the tumor only cnvpytor plots as well 👍

**Review by mathiasbio — APPROVED** [review]
Looks good to me! Nice job! ❤️

File: BALSAMIC/snakemake_rules/variant_calling/somatic_sv_tumor_normal.rule (hunk 1 above)
Comment (ivadym) [reply BALSAMIC/snakemake_rules/variant_calling/somatic_sv_tumor_normal.rule:197]: Exactly! The deliver function skips them if they are marked as temp and it doesn't include them in the .hk file. 

Regading hermes, I'm working on it now (I'll also update the validation report so we test that these files are stored)

Comment (ivadym) [reply BALSAMIC/snakemake_rules/variant_calling/somatic_sv_tumor_normal.rule:197]: Thanks for the review 🙏

Comment (ivadym) [reply BALSAMIC/snakemake_rules/variant_calling/somatic_sv_tumor_normal.rule:197]: Hermes commit: https://github.com/Clinical-Genomics/hermes/pull/86/commits/a55e390e5f6c5d36beed633032416a687acbbd7a

**Pull request merged**